Automattic/wp-calypso
# Gutenboarding: upgraded design screenshots

#### Changes proposed in this Pull Request

Re-ran the script to generate screenshots (while testing https://github.com/Automattic/wp-calypso/pull/43559).

To review, I recommend some tool such as [Github desktop](https://github.com/desktop/) that shows image differences side-by-side. Sadly Github itself doesn't. :-(

Ah screw that, I'm just gonna screenshot all these:

### Preview links, screenshots and notes for each design
<details>
  <summary>See screenshots (these are outdated, see latest comments for fresh ones</summary>

[Vesta](https://public-api.wordpress.com/rest/v1/template/demo/mayland/vesta?font_base=Raleway&font_headings=Cabin&site_title=Vesta)
<img width="856" alt="Screenshot 2020-06-23 at 14 14 28" src="https://user-images.githubusercontent.com/87168/85397225-e19c7500-b55b-11ea-8abb-878cba35163b.png">
Bolded title (this is in many designs so I'm not gonna mention it again)

[Balasana](https://public-api.wordpress.com/rest/v1/template/demo/balasana/balasana?font_base=Chivo&font_headings=Open%20Sans&site_title=Balasana)
<img width="456" alt="Screenshot 2020-06-23 at 12 44 59" src="https://user-images.githubusercontent.com/87168/85389281-e6f3c280-b54f-11ea-99c4-9ccb37679263.png">
Different spacing

[Barnsbury](https://public-api.wordpress.com/rest/v1/template/demo/barnsbury/barnsbury?font_base=Chivo&font_headings=Open%20Sans&site_title=Barnsbury)
<img width="453" alt="Screenshot 2020-06-23 at 12 44 21" src="https://user-images.githubusercontent.com/87168/85389285-e824ef80-b54f-11ea-87c5-43171fa52d89.png">
Underlined title, side-spacing gone.

[Bowen](https://public-api.wordpress.com/rest/v1/template/demo/coutoire/bowen?font_base=Fira%20Sans&font_headings=Playfair%20Display&site_title=Bowen)
<img width="458" alt="Screenshot 2020-06-23 at 12 43 56" src="https://user-images.githubusercontent.com/87168/85389287-e8bd8600-b54f-11ea-9895-62350fefc7a9.png">
Underline

[Coutoire](https://public-api.wordpress.com/rest/v1/template/demo/coutoire/coutoire?font_base=Fira%20Sans&font_headings=Playfair%20Display&site_title=Coutoire)
<img width="471" alt="Screenshot 2020-06-23 at 12 43 02" src="https://user-images.githubusercontent.com/87168/85389293-e9561c80-b54f-11ea-867d-26185a64359f.png">

[Doyle](https://public-api.wordpress.com/rest/v1/template/demo/alves/doyle?font_base=Fira%20Sans&font_headings=Playfair%20Display&site_title=Doyle)
<img width="442" alt="Screenshot 2020-06-23 at 12 42 43" src="https://user-images.githubusercontent.com/87168/85389294-e9eeb300-b54f-11ea-8a27-e9c6911a2845.png">
Side margin issue.

[Easley](https://public-api.wordpress.com/rest/v1/template/demo/maywood/easley?font_base=Roboto&font_headings=Space%20Mono&site_title=Easley)
<img width="464" alt="Screenshot 2020-06-23 at 12 42 19" src="https://user-images.githubusercontent.com/87168/85389297-e9eeb300-b54f-11ea-98d3-4370a101220e.png">
Side margin issue.

[Edison](https://public-api.wordpress.com/rest/v1/template/demo/stratford/edison?font_base=Open%20Sans&font_headings=Chivo&site_title=Edison)
<img width="458" alt="Screenshot 2020-06-23 at 12 41 54" src="https://user-images.githubusercontent.com/87168/85389302-ea874980-b54f-11ea-9deb-175b1e553796.png">

[Rivington](https://public-api.wordpress.com/rest/v1/template/demo/rivington/rivington?font_base=Montserrat&font_headings=Arvo&site_title=Rivington)
<img width="461" alt="Screenshot 2020-06-23 at 12 40 09" src="https://user-images.githubusercontent.com/87168/85389304-ea874980-b54f-11ea-92cf-b01e9dba8a96.png">

[Rockfield](https://public-api.wordpress.com/rest/v1/template/demo/rockfield/rockfield?font_base=Fira%20Sans&font_headings=Playfair%20Display&site_title=Rockfield)
<img width="455" alt="Screenshot 2020-06-23 at 12 39 18" src="https://user-images.githubusercontent.com/87168/85389306-eb1fe000-b54f-11ea-8c91-1044d6fecfe2.png">
💥 prolly needs similar treatment to D45189-code

[Stratford](https://public-api.wordpress.com/rest/v1/template/demo/stratford/stratford?font_base=Open%20Sans&font_headings=Chivo&site_title=Stratford)
<img width="848" alt="Screenshot 2020-06-23 at 14 18 40" src="https://user-images.githubusercontent.com/87168/85397583-756e4100-b55c-11ea-8298-732f50430164.png">
Image showed up.

[Cassel](https://public-api.wordpress.com/rest/v1/template/demo/mayland/cassel?font_base=Fira%20Sans&font_headings=Playfair%20Display&site_title=Cassel)
<img width="851" alt="image" src="https://user-images.githubusercontent.com/87168/85397392-23c5b680-b55c-11ea-8d51-3336db8d5999.png">

[Camdem](https://public-api.wordpress.com/rest/v1/template/demo/maywood/camdem?font_base=Roboto&font_headings=Space%20Mono&site_title=Camdem)
<img width="853" alt="image" src="https://user-images.githubusercontent.com/87168/85397325-0bee3280-b55c-11ea-9b58-db43742172a6.png">

[Reynolds](https://public-api.wordpress.com/rest/v1/template/demo/rockfield/reynolds?font_base=Fira%20Sans&font_headings=Playfair%20Display&site_title=Reynolds)
<img width="848" alt="image" src="https://user-images.githubusercontent.com/87168/85397509-4f48a100-b55c-11ea-826a-b28c7ce2a5c3.png">

[Overton](https://public-api.wordpress.com/rest/v1/template/demo/alves/overton?font_base=Raleway&font_headings=Cabin&site_title=Overton)
<img width="845" alt="image" src="https://user-images.githubusercontent.com/87168/85397493-4952c000-b55c-11ea-90a5-ca8dbd2d74be.png">

[Brice](https://public-api.wordpress.com/rest/v1/template/demo/mayland/brice?font_base=Fira%20Sans&font_headings=Playfair%20Display&site_title=Brice)
<img width="852" alt="image" src="https://user-images.githubusercontent.com/87168/85397362-17d9f480-b55c-11ea-84a4-d5464a26f46f.png">

[Alves](https://public-api.wordpress.com/rest/v1/template/demo/alves/alves?font_base=Montserrat&font_headings=Arvo&site_title=Alves)
<img width="851" alt="image" src="https://user-images.githubusercontent.com/87168/85397342-11e41380-b55c-11ea-9383-84c203cbba32.png">

[Mayland](https://public-api.wordpress.com/rest/v1/template/demo/mayland/mayland?font_base=Cabin&font_headings=Raleway&site_title=Mayland)
<img width="851" alt="image" src="https://user-images.githubusercontent.com/87168/85397436-36d88680-b55c-11ea-85dd-ae793c0fbcb0.png">

[Maywood](https://public-api.wordpress.com/rest/v1/template/demo/maywood/maywood?font_base=Fira%20Sans&font_headings=Playfair%20Display&site_title=Maywood)
<img width="858" alt="image" src="https://user-images.githubusercontent.com/87168/85397450-3e982b00-b55c-11ea-9e02-d9d9e424b52e.png">

[Leven](https://public-api.wordpress.com/rest/v1/template/demo/leven/leven?font_base=Open%20Sans&font_headings=Chivo&site_title=Leven)
<img width="849" alt="image" src="https://user-images.githubusercontent.com/87168/85397424-30e2a580-b55c-11ea-9792-2c29f278f100.png">

[Gibbs](https://public-api.wordpress.com/rest/v1/template/demo/rockfield/gibbs?font_base=Fira%20Sans&font_headings=Fira%20Sans&site_title=Gibbs)
<img width="847" alt="image" src="https://user-images.githubusercontent.com/87168/85397413-2cb68800-b55c-11ea-89a1-e29061e5c150.png">
</details>

##### The two FSE demos we can probably ignore for now:

- [Reynolds (FSE)](https://public-api.wordpress.com/rest/v1/template/demo/seedlet-blocks/reynolds?font_base=Fira%20Sans&font_headings=Playfair%20Display&site_title=Reynolds%20%28FSE%29)
- [Vesta (FSE)](https://public-api.wordpress.com/rest/v1/template/demo/seedlet-blocks/vesta?font_base=Raleway&font_headings=Cabin&site_title=Vesta%20%28FSE%29)


#### Testing instructions

See designs step at `/new`


@ianstewart could you look these through and see if some of the changes are problems or is everything expected? I didn't have time to look into previews nor real sites yet if the problems are present in those too.
